markdown/preface.markdown

```diff
@@ -434,9 +429,20 @@ Note that it is common etiquette to *subscribe* to the list before posting.
 If you don't, it usually means someone will have to manually check that your email isn't spam before allowing it to be posted to the list.
 It may seem annoying to join yet another mailing list, but we highly recommend it: it is a great place to learn!
 
+## Installing Python - Anaconda
+
+Throughout this book we’re going to assume that you have Python 3.5 (or later) and have all the major SciPy packages installed.
+We list all of the requirements and the versions we used in the **environment.yml** file packaged with the data for this book.
+The easiest way to get all of these components is to install the Anaconda Python distribution, which includes all the requirements as well as many more scientific Python packages.
+You can download Anaconda here: http://continuum.io/downloads. You will also find detailed installation instructions.
```

Comment:
Which bit to we stick with? Miniconda + environments.yml seems safest in case we end up using something that's not in Anaconda (although I guess that's unlikely?).
Anaconda is probably easiest as it's a single download, particularly if they have to install 3.5.